modules/xfem/src/userobjects/MeshCut2DFractureUserObject.C

```diff
@@ -63,40 +82,60 @@ MeshCut2DFractureUserObject::initialize()
 void
 MeshCut2DFractureUserObject::findActiveBoundaryGrowth()
 {
-  const VectorPostprocessorValue & k1 = getVectorPostprocessorValueByName(
-      "II_KI_" + _ring_number_string, "II_KI_" + _ring_number_string);
-  const VectorPostprocessorValue & k2 = getVectorPostprocessorValueByName(
-      "II_KII_" + _ring_number_string, "II_KII_" + _ring_number_string);
-
   // k1 is empty on the very first time step because this UO is called before the
-  // InteractionIntegral vpp
-  if (k1.empty())
+  // InteractionIntegral or crackFrontStress vpp
+  if (_ki_vpp.empty() && _stress_vpp.empty())
```

Comment:
If you're hitting this, it seems like these aren't getting run in the right order. Let's check that.